### ISSUE TITLE
Error calling entry remove callback Telenet Telemeter for telenet_telemeter

### ISSUE THREAD

**Issue by tjorim** [myTselection/telenet_telemeter#17]
After trying to add the integration again for checking #15, I wanted to remove it. Getting the following error:

```
Logger: homeassistant.config_entries
Source: config_entries.py:1419
First occurred: 06:28:24 (2 occurrences)
Last logged: 06:28:28

Error calling entry remove callback Telenet Telemeter for telenet_telemeter
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 579, in async_remove
    await component.async_remove_entry(hass, self)
  File "/config/custom_components/telenet_telemeter/__init__.py", line 89, in async_remove_entry
    await hass.config_entries.async_forward_entry_unload(config_entry, PLATFORMS)
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 1419, in async_forward_entry_unload
    if domain not in self.hass.config.components:
TypeError: unhashable type: 'list'
```

**Comment by myTselection** [comment]
is this still the case with 0.9.4? the sensor have now unique names which may resolve this.

**Comment by myTselection** [comment]
fixed as of 1.1.0
